src/olympia/devhub/forms.py

```diff
@@ -561,7 +561,7 @@ def clean(self):
                     elif version.unreviewed_files:
                         next_url = reverse('devhub.submit.version.details',
                                            args=[self.addon.slug, version.pk])
-                        msg = jinja2.Markup('%s <a href="%s">%s</a>' % (
+                        msg = DoubleSafe('%s <a href="%s">%s</a>' % (
```

Comment:
I've toyed with the idea of replacing `jinja2.Markup` with `DoubleSafe` everywhere to be safe.